tests/test_unittest/test_gtscript_frontend.py

```diff
@@ -148,7 +188,7 @@ def definition_func(inout_field: gtscript.Field[float]):
             with computation(PARALLEL), interval(...):
                 inout_field = inout_field[0, 0, 0] + WRONG_VALUE_CONSTANT
 
-        with pytest.raises(gt_frontend.GTScriptSymbolError, match=r".*WRONG_VALUE_CONSTANT.*"):
+        with pytest.raises(gt_frontend.GTScriptDefinitionError, match=r".*WRONG_VALUE_CONSTANT.*"):
             compile_definition(definition_func, "test_wrong_value", module, externals=externals)
```

Comment:
@egparedes This was the only place I could tell that the behavior changed.

Reply:
Ok, then, raising a different Exception should not matter too much.